docs/Tutorials/ConnectInfura.md

```diff
@@ -0,0 +1,108 @@
+---
+description: Connect to an Infura endpoint
+---
+
+# Connect to an Infura endpoint
+
+This tutorial walks you through connecting Quorum Key Manager (QKM) to an Infura endpoint.
+
+This tutorial connects to Infura to allow a QKM-managed account to interact with a smart contract on the Rinkeby network.
```

Comment:
This makes it sound like the tutorial is actually connecting to Infura. Maybe something like:
```suggestion
This tutorial demonstrates connecting to Infura to allow a QKM-managed account to interact with a smart contract on the Rinkeby network.
```